facade/src/main/java/org/jboss/pnc/facade/providers/BuildConfigurationProviderImpl.java

```diff
@@ -94,6 +94,9 @@
     @Inject
     private SCMRepositoryProvider scmRepositoryProvider;
 
+    @Inject
+    private BuildConfigRevisionHelper buildConfigRevisionHelper;
+
```

Comment:
IMHO we should either inject the constructor or inject the fields. Combining both approaches looks confusing to me.